ui/ducks/send/send.js

```diff
@@ -2740,15 +2740,15 @@ export function useMyAccountsForRecipientSearch() {
  * @returns {ThunkAction<void>}
  */
 export function resetRecipientInput() {
-  return async (dispatch, getState) => {
+  return (dispatch, getState) => {
     const state = getState();
     const chainId = getCurrentChainId(state);
     showLoadingIndication();
-    await dispatch(addHistoryEntry(`sendFlow - user cleared recipient input`));
-    await dispatch(updateRecipientUserInput(''));
-    await dispatch(updateRecipient({ address: '', nickname: '' }));
-    await dispatch(resetDomainResolution());
-    await dispatch(validateRecipientUserInput({ chainId }));
+    dispatch(addHistoryEntry(`sendFlow - user cleared recipient input`));
```

Comment:
Why do we not need to await here anymore?

Reply:
I'm not sure why we were in the first place? None of these actions are async. Regardless, this was causing an async issue where the send page was using old state, specifically line 103 in `Recipient.tsx`: `userInput.length > 0 ? CONTACTS_TAB_KEY : ACCOUNTS_TAB_KEY`

Reply:
Looks like `updateRecipient` is async?

Reply:
Woops didn't realize, well awaiting just that seems to be fine as well. Pushing an update.